parse/DoubleValueRefParser.cpp

```diff
@@ -117,25 +120,39 @@ parse::double_parser_rules::double_parser_rules(
         =   int_complex_grammar [ _val = construct_movable_(new_<ValueRef::StaticCast<int, double>>(deconstruct_movable_(_1, _pass))) ]
         ;
 
+    named_real_valueref
+        = (     tok.Named_ >> tok.Real_
+             >  label(tok.Name_) > tok.string
+             >  label(tok.Value_) > primary_expr.alias()
```

Comment:
not sure what the `alias` call is doing here... is it needed?

Reply:
alias is doing a copy of the structure AFAIU. Using primary_expr only at that time did not work. Can try again.

Reply:
works without alias()